modules/olm-mirroring-catalog.adoc

```diff
@@ -98,97 +94,32 @@ src image has index label for database path: /database/index.db
-operatorbundle_name = jaeger-operator.v1.17.6
-
-image = {example-registry}/distributed-tracing/jaeger-es-index-cleaner-rhel7@sha256:c64ac461d96523516a199bd132ad4d7148317e503a735028f0d8f7ba063a61cb
-operatorbundle_name = jaeger-operator.v1.17.6
-
-image = {example-registry}/distributed-tracing/jaeger-rhel7-operator:1.13.2
-operatorbundle_name = jaeger-operator.v1.13.2-1
-----
-
-... Use the results from the previous step to help you edit the `mapping.txt` file to only include the subset of images you want to mirror.
-+
-For example, you can use the `image` values from the previous example output to find that the following matching lines exist in your `mapping.txt` file:
-+
-.Matching image mappings in `mapping.txt`
-[source,terminal,subs="attributes+"]
-----
-...
-{example-registry}/distributed-tracing/jaeger-all-in-one-rhel7@sha256:41f769c2c32f3f050aa42d86f084b739914ff9ba2f0aed2d9b0b69357b48459d=quay.io/adellape/distributed-tracing-jaeger-all-in-one-rhel7:5cf7a033
-...
-{example-registry}/distributed-tracing/jaeger-es-index-cleaner-rhel7@sha256:c64ac461d96523516a199bd132ad4d7148317e503a735028f0d8f7ba063a61cb=quay.io/adellape/distributed-tracing-jaeger-es-index-cleaner-rhel7:ecfd2ca7
-...
-{example-registry}/distributed-tracing/jaeger-rhel7-operator:1.13.2=quay.io/adellape/distributed-tracing-jaeger-rhel7-operator:1.13.2
-...
-----
 +
-In this example, if you only want to mirror these images, you would then remove all other entries in the `mapping.txt` file and leave only the above matching image mapping lines.
-
-.. Still on your workstation with unrestricted network access, use your modified `mapping.txt` file to mirror the images to your registry using the `oc image mirror` command:
-+
-[source,terminal,subs="attributes+"]
-----
-$ oc image mirror \
-    [-a ${REG_CREDS}] \
-    -f ./{index-image}-manifests/mapping.txt
-----
-
-. Apply the `ImageContentSourcePolicy` object:
-+
-[source,terminal,subs="attributes+"]
-----
-$ oc apply -f ./{index-image}-manifests/imageContentSourcePolicy.yaml
-----
+[IMPORTANT]
+====
+If you used the `--manifests-only` flag in this step and want to further trim the subset of packages to be mirrored, see the steps in the _Mirroring a Package Manifest Format catalog image_ procedure about modifying your `mapping.txt` file and using the file with the `oc image mirror` command. After following those further actions, you can continue this procedure.
+====
 
-. If you are not using a custom, pruned version of an index image, push the Red Hat-provided index image to your registry:
+. Create the `ImageContentSourcePolicy` object:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ oc image mirror \
-    [-a ${REG_CREDS}] \
-    {index-image-pullspec} \//<1>
-    <mirror_registry>:<port>/<namespace>/{index-image}:{tag} <2>
+$ oc create -f ./{index-image}-manifests/imageContentSourcePolicy.yaml
```

Comment:
It should be `oc create -f ./manifests-<index-image-name>-<random_number>/imageContentSourcePolicy.yaml`